application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -34,7 +34,7 @@
   <date>1571915591000</date>
   <contentUpdateDate>1571915591000</contentUpdateDate>
   <version>1.1</version>
-  <title>#if("$!doc.getObject('XPoll.XPollClass').getProperty('name').value" != '')$!doc.getObject('XPoll.XPollClass').getProperty('name').value #elseif($doc.documentReference.name=='XPollSheet' )Polls Sheet #else${doc.title} #end</title>
+  <title>#if("$!doc.getObject('XPoll.XPollClass').documentReference.name" != '')$!doc.getObject('XPoll.XPollClass').documentReference.name #elseif($doc.documentReference.name=='XPollSheet' )Polls Sheet #else${doc.title} #end</title>
```

Comment:
There is no `getDocumentReference()` in the Object you get from document. I think this is not the source of your deprecation usage.